File: src/main/java/kr/mjc/rachel/spring/jdbc/user/ListUsers.java
package kr.mjc.rachel.spring.jdbc.user;

import kr.mjc.rachel.basics.jdbc.user.dao.UserDao;
import kr.mjc.rachel.spring.jdbc.AppTest;
import lombok.extern.slf4j.Slf4j;

@Slf4j
public class ListUsers {
  public static void main(String[] args) {
    new AppTest().test(UserDao.class, (userDao, log) -> {
      log.info(userDao.listUsers(5, 1).toString());
    });
  }
}

File: src/main/kotlin/kr/mjc/jacob/spring/jdbc/user/ListUsers.kt
package kr.mjc.jacob.spring.jdbc.user

import kr.mjc.jacob.basics.jdbc.Limit
import kr.mjc.jacob.basics.jdbc.user.dao.UserDao
import kr.mjc.jacob.spring.jdbc.AppTest
import java.util.*

fun main() {
  AppTest().test(UserDao::class.java) { userDao, log ->
    print("List - count page ? ")
    val limit = Limit().apply {
      Scanner(System.`in`).use {
        count = it.nextInt()
        page = it.nextInt()
      }
    }
    val userList = userDao.listUsers(limit)
    log.debug(userList.toString())
  }
}